pilgrimor/abc/migrator.py

```diff
@@ -100,25 +100,20 @@ def run_migrations(  # noqa: C901
             command = "apply"
         else:
             command = "rollback"
-        success_migrations: List[str] = []
-
-        for migration in migrations:
-            try:
-                if apply and version:
-                    self._apply_migration(migration=migration, version=version)
-                elif not apply:
-                    self._rollback_migration(migration=migration)
-            except Exception as exc:
-                not_applied_migrations = set(migrations) - set(success_migrations)
-                sys.exit(
-                    f"Can't {command} migrations {not_applied_migrations} because "
-                    f"there is error - {exc}",
-                )
-            print(success_text(f"Command {command} done for {migration}"))
-            success_migrations.append(migration)
+
+        try:
+            if apply and version:
+                self._apply_migrations(migrations=migrations, version=version)
+            elif not apply:
+                self._rollback_migrations(migrations=migrations)
+        except Exception as exc:
+            sys.exit(
```

Comment:
This should be as `error_message`

Reply:
this is not my code, it was already there before )

Reply:
Please add) Thats my bad)